client/src/components/User/DiskUsage/Management/services.js

```diff
@@ -0,0 +1,119 @@
+import axios from "axios";
+import { getAppRoot } from "onload/loadConfig";
+import { rethrowSimple } from "utils/simple-error";
+import { CleanableSummary, CleanupResult } from "./Cleanup/model";
+
+const datasetKeys = "id,name,size,update_time,hda_ldda";
+const isDataset = "q=history_content_type-eq&qv=dataset";
+const isDeleted = "q=deleted-eq&qv=True";
+const isNotPurged = "q=purged-eq&qv=False";
+const maxItemsToFetch = 5000;
```

Comment:
I think that can be very heavy on the server. I'd either leave this unspecified and adjust the message to say "more than 500 items" or I'd limit this at a lower number and return the total matches in the header.

Reply:
Understand!, I will limit it back to 500 and indicate this limit in the operation description so the user doesn't get confused if more items appear after the first pass. Since it's ordered by size by default it should be good enough to review/bulk delete in chunks of 500.

Reply:
Ok, the limit is now back to 500 and the review dialog will display `500+` and a clarification message in the footer when there are at least 500 items

![Screenshot from 2022-03-25 13-15-15](https://user-images.githubusercontent.com/46503462/160119657-f600740b-9bfc-44b3-b366-30cc24e129ff.png)